src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -459,7 +459,7 @@ public byte Updatability
 
         public bool IsReadOnly
         {
-            get => flags.HasFlag(_SqlMetadataFlags.IsReadOnlyMask);
+            get => (flags & _SqlMetadataFlags.IsReadOnlyMask) == 0;
```

Comment:
Curious: How is this change different than flags.HasFlag and why do the other properties the use flags.HasFlag work when IsReadOnly doesn't?